src/Uno.UWP/Storage/Helpers/StorageFileHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno;
+
+namespace Windows.Storage.Helpers;
+
+internal partial class StorageFileHelper
+{
+	/// <summary>
+	/// Determines if an asset or resource exists within application package
+	/// </summary>
+	/// <param name="fileName">relative file path</param>
+	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
```

Comment:
```suggestion
	/// <summary>
	/// Determines if an asset or resource exists within the application package
	/// </summary>
	/// <param name="fileName">relative file path</param>
	/// <returns>A task that will complete with a result of true if the file exists, otherwise with a result of false.</returns>
```